server.py

```diff
@@ -49,32 +58,36 @@ async def get_garage_data():
         sqlhelper.insert_garage_data(DB_FILE, garage, garage_data[garage][0], timestamp)
         sqlhelper.delete_garage_data(DB_FILE, garage)
 
+    logger.debug("Garage data updated successfully")
     return garage_data
 
-logging.Formatter.converter = time.gmtime
-logging.basicConfig(
-    format="%(asctime)s.%(msecs)03dZ %(levelname)s:%(name)s:%(message)s",
-    datefmt="%Y-%m-%dT%H:%M:%S",
-    level= logging.ERROR - (args.verbose*10),
-)
-
 @app.get("/")
 async def root():
     return "Welcome to SJSU Parking!"
 
+@app.get("/get_data/{garage}")
+async def get_data(garage: str):
+    logger.debug(f"Getting {garage} data")
+    data = sqlhelper.get_garage_data(DB_FILE, garage)
+
+    if data:
+        return {"Garage": garage, "data": data}
+    else:
+        return {"No data found for": garage}
+
 def helper_thread():
-    print("Helper thread started.")  
+    logger.debug("Helper thread started.")  
     while True:
         current_time = datetime.now(pytz.timezone('US/Pacific'))
-        print(f"Current time: {current_time}")
-        if current_time.hour >= 8 and current_time.hour < 14:
+        logger.info(f"Current time: {current_time}")
+        if current_time.hour >= 8 and current_time.hour < 18:
             try:
                 # Between 8am-2pm, call endpoint
                 asyncio.run(get_garage_data())  
             except Exception as e:
-                print(f"An error occurred: {e}")
+                logger.error(f"An error occurred: {e}")
```

Comment:
you can just do 
```py
except Exception:
    logger.exception("unable to get garage data")
```

and the exception will be included in your log
https://stackoverflow.com/a/5191885